vendor/assets/javascripts/webchat.js

```diff
@@ -94,7 +94,7 @@ function sWOTrackPage() {
       sWOChatElement = document.getElementById('whoson_chat_link');
     }
     sWOUrl += "&response=g";
-    sWOChatstart += "?domain=" + sWODomain;
+    -1 == sWOChatstart.indexOf("domain") && (sWOChatstart += "?domain=" + sWODomain)
```

Comment:
don't think so, it's already been fully tested on UAT, this edit is recommended by the web chat supplier to stop the site monitoring timing out, although I would prefer one of the FE team to review and sign off on it though